src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -205,22 +205,14 @@ protected function isCircularReference($object, &$context)
      *
      * @final since Symfony 4.2
```

Comment:
Should we make this method final for real? considering https://github.com/symfony/symfony/pull/27020#discussion_r184028869

> the proper way to extend the handling of circular references is to register a handler, not to extend the method in child classes.

Reply:
We don't have to.